beeflow/client/bee_client.py

```diff
@@ -312,6 +314,51 @@ def is_parent(parent, path):
     if os.path.commonpath([os.path.realpath('/var/tmp'), workdir]) == os.path.realpath('/var/tmp'):
         error_exit("Workflow working directory cannot be in \"/var/tmp\"")
 
+    # TODO: this should be in a function somewhere
```

Comment:
I tried to find some kind of python package that would do this logic, but didn't come up with anything.

I'm not sure the best place to put this function. It should likely also have some tests.